pkg/LinuxGeneralModule/lgm.go

```diff
@@ -363,7 +361,12 @@ func Initialize() {
 	brTenant = "br-tenant"
 	ipMtu = config.GlobalConfig.LinuxFrr.IPMtu
 	ctx = context.Background()
-	nlink = utils.NewNetlinkWrapperWithArgs(config.GlobalConfig.Tracer)
+	RouteTableGen = utils.IDPoolInit("RTtable", routingTableMin, routingTableMax)
```

Comment:
`IDPoolInit` can just return `error` if failed to initialize

Reply:
added bool return for the initialization status